certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -1390,7 +1390,7 @@
             "type": "string",
             "description": "Verification Type",
             "x-ms-summary": "Verification Type",
-            "enum":["None", "Access Code", "ID Verification", "Phone Call", "SMS","Knowledge Based"],
```

Comment:
Are you removing SMS authentication?

Reply:
Never mind. I just read through the slack conversation on Phone auth being used for both phone call and SMS. In that case SMS feature needs a regress testing once this is ready for testing.

Reply:
Though the SMS auth code was there, we had never enabled it in the Microsoft official dev branch. So, nothing to regress on the SMS auth.